### PR TITLE
chore: rename regenBridge to bridgeController

### PR DESCRIPTION
**Pull request by kargakis**
Addressing https://github.com/regen-network/toucan-bridge/pull/9#discussion_r906000719

### CODE REVIEW

File: contracts/ToucanRegenBridge.sol [hunk 1]
```diff
@@ -53,12 +53,12 @@ contract ToucanRegenBridge is Ownable, Pausable {
     // ----------------------------------------
 
     /**
-     * @dev Sets the values for {regenBridge} and {toucanContractRegistry}.
+     * @dev Sets the values for {bridgeController} and {toucanConbegistry}.
```
Comment (pheuberger) [inline contracts/ToucanRegenBridge.sol]: typo `toucanConbegistry`

**Review by marceljay — APPROVED** [review]
LFG

**Comment by kargakis** [comment]
@marceljay sorry, had to push again to fix a comment, ptal

**Review by pheuberger — COMMENTED** [review]
lgtm!

File: contracts/ToucanRegenBridge.sol (hunk 1 above)
Comment (kargakis) [reply contracts/ToucanRegenBridge.sol]: Heh, already addressed ^_^

Comment (pheuberger) [reply contracts/ToucanRegenBridge.sol]: haha, nice! 💪

**Pull request merged**